apps/services/form-system/src/app/modules/applications/applications.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Controller, Get, Param } from '@nestjs/common'
+import { ApiTags } from '@nestjs/swagger'
+import { ApplicationsService } from './applications.service'
+import { ApplicationDto } from './models/dto/application.dto'
+import { Documentation } from '@island.is/nest/swagger'
+
+@ApiTags('applications')
+@Controller('applications')
```

Comment:
FYI you can version your controller like this:

```
@Controller({
  path: 'applications',
  version: ['1', VERSION_NEUTRAL],
})
```

`VERSION_NEUTRAL` indicates that this will work for any version passed in the request, or no version.